pkg/resource/google/google_compute_instance_test.go

```diff
@@ -17,6 +18,16 @@ func TestAcc_Google_ComputeInstance(t *testing.T) {
 		},
 		Checks: []acceptance.AccCheck{
 			{
+				// New resources are not visible immediately through GCP API after an apply operation.
+				// Logic below retries driftctl scan using a back-off strategy of retrying 'n' times
+				// and doubling the amount of time waited after each one.
+				ShouldRetry: func(result *test.ScanResult, retryDuration time.Duration, retryCount uint8) bool {
```

Comment:
Can we extract this func some where, to avoid the duplication of its logic and the explanatory comment everywhere?